### PR TITLE
Rollback to substrate tag fragnova-v0.0.6

### PR DESCRIPTION
**Pull request by alebaffa**
Rollback needed after IPFS issues [introduced](https://github.com/fragcolor-xyz/substrate/commit/50325f1f0b36210d1730f7c903ab89083fe37a05) with Substrate tag `fragnova-v0.29.1`.

Rolled back to this tag where IPFS works with `go-ipfs_v0.12.0`.

### CODE REVIEW

**Comment by alebaffa** [comment]
All functionalities introduced in our pallets since then remained untouched, obviously.

**Comment by sinkingsugar** [comment]
Not sure I get it, is this a backport?
New devel functionalities are IN or not?
Also does IPFS test work fully?

**Comment by alebaffa** [comment]
I moved back to `fragnova-v0.0.6` tag, so keeping all latest functionalities from `devel`. 
IPFS works fully, I tested manually with go-ipfs_v0.12.0.

**Comment by sinkingsugar** [comment]
> I moved back to `fragnova-v0.0.6` tag, so keeping all latest functionalities from `devel`. IPFS works fully, I tested manually with go-ipfs_v0.12.0.

Nice, are the tests also fixed? the ipfs test

File: shards/test-ipfs.sh [hunk 1]
```diff
@@ -4,9 +4,9 @@ set -e
 # INSTALL IPFS
 apt-get update
 apt-get install -y wget
-wget -q https://dist.ipfs.tech/kubo/v0.16.0/kubo_v0.16.0_linux-amd64.tar.gz
-tar -xvzf kubo_v0.16.0_linux-amd64.tar.gz
-export PATH=$PATH:`pwd`/kubo
+wget -q https://dist.ipfs.tech/go-ipfs/v0.12.0/go-ipfs_v0.12.0_linux-amd64.tar.gz
```
Comment (sinkingsugar) [inline shards/test-ipfs.sh:7]: kubo 0.17, let's do a try with that too? or doesn't work?

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -210,13 +215,27 @@ parameter_types! {
...
+		.for_class(DispatchClass::Operational, |weights| {
+			weights.max_total = Some(MAXIMUM_BLOCK_WEIGHT);
+			// Operational transactions have some extra reserved space, so that they
+			// are included even if block reached `MAXIMUM_BLOCK_WEIGHT`.
+			weights.reserved = Some(
+				MAXIMUM_BLOCK_WEIGHT - NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT
+			);
+		})
+		.avg_block_initialization(AVERAGE_ON_INITIALIZE_RATIO)
+		.build_or_panic();
```
Comment (sinkingsugar) [inline runtime/src/lib.rs:238]: let's make sure this is correct

**Review by sinkingsugar — APPROVED** [review]
Approved with some comments!

File: shards/test-ipfs.sh (hunk 1 above)
Comment (alebaffa) [reply shards/test-ipfs.sh:7]: It doesn't work with kubo 0.17.

File: runtime/src/lib.rs (hunk 2 above)
Comment (alebaffa) [reply runtime/src/lib.rs:238]: Yes, it is the version there was back then.
The [new version](https://github.com/paritytech/substrate/blob/ea3ca3f757ff9d9559665719a77da81f4cf0f0ce/bin/node-template/runtime/src/lib.rs#L138), used also in latest node-template, is a refactoring that extracts this calculation into `fn with_sensible_defaults` in [frame/system/limits.rs](https://github.com/paritytech/substrate/blob/ea3ca3f757ff9d9559665719a77da81f4cf0f0ce/frame/system/src/limits.rs#L323). In addition, the new version uses a utility function `set_proof_size`, which has been introduced in `weight_v2` even [after](https://github.com/paritytech/substrate/commit/17c07af0b953b84dbe89341294e98e586f9b4591) the substrate tag that introduced the problems with ipfs.

File: shards/test-ipfs.sh (hunk 1 above)
Comment (sinkingsugar) [reply shards/test-ipfs.sh:7]: That's so weird to be honest but must be libp2p incompatibilities somehow ( a library engineered for compatibility 😄 )

**Pull request merged**